src/models/App.js

```diff
@@ -40,6 +40,27 @@ export const schema = new Schema({
       type: 'String',
     },
   },
+
+  // Hipchat webhooks
+  webhooks: [Number],
```

Comment:
About to move this webhooks key onto the hipchat object. I'm starting to think it might make sense to just create a hipchat model. What do you think?